sql/delete.go

```diff
@@ -18,14 +18,36 @@ package sql
 
 import (
 	"bytes"
+	"fmt"
 
+	"github.com/cockroachdb/cockroach/client"
 	"github.com/cockroachdb/cockroach/roachpb"
 	"github.com/cockroachdb/cockroach/sql/parser"
 	"github.com/cockroachdb/cockroach/sql/privilege"
 	"github.com/cockroachdb/cockroach/util/log"
 )
 
-// Delete deletes rows from a table.
+type deleteNode struct {
```

Comment:
judging by the commit message, there are a lot of changes in this commit that were not intended.